website/source/docs/providers/aws/r/app_cookie_stickiness_policy.html.markdown

```diff
@@ -15,20 +15,20 @@ Provides an application cookie stickiness policy, which allows an ELB to wed its
 ```
 resource "aws_elb" "lb" {
     name = "test-lb"
-	  availability_zones = ["us-east-1a"]
-	  listener {
-	      instance_port = 8000
-		    instance_protocol = "http"
-		    lb_port = 80
-		    lb_protocol = "http"
-	  }
+	availability_zones = ["us-east-1a"]
+	listener {
+	  instance_port = 8000
+	  instance_protocol = "http"
+	  lb_port = 80
+	  lb_protocol = "http"
+    }
 }
 
 resource "aws_app_cookie_stickiness_policy" "foo" {
-	  name = "foo_policy"
-	  load_balancer = "${aws_elb.lb.id}"
-	  lb_port = 80
-	  cookie_name = "MyAppCookie"
+	name = "foo_policy"
```

Comment:
Well, you didn't cause this by your changes :smiley: but the example has invalid name for some reason, so I think we can fix it.